types/stubs/google/auth/__init__.pyi

```diff
@@ -1,6 +1,6 @@
 from google.auth.credentials import Credentials
-from typing import Tuple
+from typing import Tuple, Iterable
 
 
-def default() -> Tuple[Credentials, str]:
+def default(scopes: Iterable[str]) -> Tuple[Credentials, str]:
```

Comment:
This line updates our internal typestubs to include the additional supported parameter to this method.